### PR TITLE
Bumped sirupsen/logrus from 1.4.2 to 1.6.0

### PR DESCRIPTION
**Pull request by mvisonneau**
- Amended `go.mod` to bump `sirupsen/logrus` from **1.4.2** to **1.6.0**
- Ran `go mod vendor`

```release-note
Bumped sirupsen/logrus from 1.4.2 to 1.6.0
```

### CODE REVIEW

**Comment by aanm** [comment]
FYI https://github.com/cilium/cilium/pull/11149#pullrequestreview-400297870

I'll double check if 1.6.0 fixed those issues.

**Review by joestringer — APPROVED** [review]
Changes LGTM from janitors codeowner. Will defer to @aanm on @cilium/vendor / followup on whether this addresses any known issues.

**Review by aanm — CHANGES_REQUESTED** [review]
It does not seem to address the concerns found in https://github.com/cilium/cilium/pull/11149#pullrequestreview-400297870 perhaps it would be better to wait for a release with that bug fixed,

**Pull request closed without merging**
